src/component/common/util.js

```diff
@@ -114,32 +113,3 @@ export const modalStyles = {
         transform: 'translate(-50%, -50%)',
     },
 };
-
-export const updateIndexInArray = (array, index, newValue) =>
-    array.map((v, i) => (i === index ? newValue : v));
-
-export const showPnpsFeedback = user => {
-    if (!user) return;
-    if (!user.feedback) return;
-    if (user.feedback.length > 0) {
-        const feedback = user.feedback.find(
-            feedback => feedback.feedbackId === 'pnps'
-        );
-
-        if (!feedback) return false;
-
-        if (feedback.neverShow) {
-            return false;
-        }
-
-        if (feedback.given) {
```

Comment:
I think I have forgotten to add this check to the Feedback component itself when refactoring. The intention is that if you have given feedback we should start showing the component again after six months. I think it's current state is to return null if given is true. Could we add that to this PR?